### PR TITLE
Add info about `index` config to notify documentation

### PR DESCRIPTION
**Pull request by plasota**
## Proposed change
There is an undocumented `index` configuration in the `notify` section of the ecobee integration. For anyone with multiple Ecobee thermostats, you need to set the index for the notification to go to anything other than the 0th index thermostat (the default value). This PR adds documentation around this config.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Link to undocumented `index` config in ecobee's `notify.py`: https://github.com/home-assistant/core/blob/0c04ca20c6571635860a4d1fc1618bf58b92797f/homeassistant/components/ecobee/notify.py#L10

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @plasota, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19722).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/ecobee.markdown [hunk 1]
```diff
@@ -90,6 +90,7 @@ To use this notification platform in your installation, add the following to you
 notify:
   - name: NOTIFIER_NAME
     platform: ecobee
+    index: 0
```
Comment (frenck) [inline source/_integrations/ecobee.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ecobee.markdown (hunk 1 above)
Comment (plasota) [reply source/_integrations/ecobee.markdown]: Makes sense, made the requested change

**Review by frenck — APPROVED** [review]
Thanks, @plasota 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**